src/ex3/ItemManager.js

```diff
@@ -0,0 +1,171 @@
+import PokemonClient from "./PokemonClient.js";
+import {promises as fs} from "fs";
+import asciifyImage from 'asciify-image';
+import chalk from "chalk";
+export default class ItemManager {
+  constructor() {
+    this.taskList = [];
+    this.pokemonClinet = new PokemonClient();
+
+  }
+
+  async addItem(item) {
+    try{
+      let data =  await fs.readFile("todoDB.json")
```

Comment:
ItemManager should only handle the tasks logic. it's not responsible for reading and writing from files. please create new class handling the reading and writing.
by separating our logic this way, our code is open to changes, which means, changes can be made easily.
Imagine that now you have to write to DB instead of a file. all you have to do is to implement the exact same methods in a DBManager class for example, and only replace the implementation of each method to write to the DB instead of a file.